lib/ibm_power_hmc/connection.rb

```diff
@@ -213,7 +213,7 @@ def lpar_profiles(lpar_uuid)
     # @param params [Hash] Job parameters.
     # @param sync [Boolean] Start the job and wait for its completion.
     # @return [IbmPowerHmc::HmcJob] The HMC job.
-    def poweron_lpar(lpar_uuid, params = {}, sync = true)
+    def poweron_lpar(lpar_uuid, params = {}, sync: true)
```

Comment:
Would this require changes to callers in manageiq-providers-ibm_power_hmc?

Reply:
@dberg1 That's a good question, I haven't tested this feature yet. I will try to work on it today to test if it works.